requirements.txt

```diff
@@ -2,7 +2,7 @@
 # - tests/unit/constraints.txt
 # - tests/integration/constraints.txt
 # - tests/integration/targets/setup_botocore_pip
-botocore>=1.19.0
-boto3>=1.16.0
+botocore>=1.23.0
```

Comment:
See PR comments. I had to do this for the test to run. boto3 version 1.16 does not have the `opensearch` service.